control/trajectory_follower/include/trajectory_follower/mpc_lateral_controller.hpp

```diff
@@ -133,6 +134,9 @@ class TRAJECTORY_FOLLOWER_PUBLIC MpcLateralController : public LateralController
   double m_ego_nearest_dist_threshold;
   double m_ego_nearest_yaw_threshold;
 
+  // debug marker
+  visualization_msgs::msg::MarkerArray m_debug_marker_array;
```

Comment:
I don't think adding a member variable which can be a local variable is a good way.
What do you think?

Reply:
I made it as a member for considering adding other markers later. 
But in this PR, only the maker for steer converges, so it's also ok to use a local variable.

Reply:
If this variable really has to be global later, you don't have to make it local.

Reply:
No real plans, just a possibility.

Reply:
Moved these markers to interface nodes.

Reply:
so removed member markers from lateral controller.